Makefile

```diff
@@ -23,7 +23,8 @@ up-all:
 
 # starts the symfony server in a isolated environment, with the default port 8000
 serve:
-	 ${SYMFONY_BIN} serve -d
+	${SYMFONY_BIN} server:ca:install
+	${SYMFONY_BIN} serve -d
```

Comment:
Not sure what this is for. Is it needed? If so, why?

Reply:
https://symfony.com/doc/current/setup/symfony_server.html, it was mentioned for local development to include the local certificate TLS. I'm sure how to only do this once, so I've included it inside of `make dev`.

Reply:
Oh right, the warning. You can safely ignore that. The local dev env isn't accepting connections from external clients, so this is a bit redundant. The warning is a bit noisy though. Let's talk about it when we meet next